### PR TITLE
Web service

### PR DESCRIPTION
**Pull request by dmitriim**

### CODE REVIEW

File: lang/en/tool_userupsert.php [hunk 1]
```diff
@@ -26,3 +26,14 @@
...
 $string['pluginname'] = 'User upsert';
+$string['privacy:metadata'] = 'The User upsert plugin does not store any personal data.';
+$string['fields'] = 'Web service fields';
+$string['fields_desc'] = 'A list of expected user fields for the web service and their descriptions. The field name followed by a pipe (|) and the description of that field. Only one field per line, separated by a line break in between each field.
+Those fields will appear in the <a href="/admin/webservice/documentation.php">Moodle API Documentation documentation</a> for tool_userupsert_upsert_users web service.
+
+For example:
+<pre>
+userName | Username field.
+authType | Auth type of the user.
```
Comment (golenkovm) [inline lang/en/tool_userupsert.php:37]: Are the full stops expected in this example? I don't know how the description will be displayed later.

**Comment by golenkovm** [comment]
If you've finished with this PR I'm happy to review all the bits in this PR and close #1 #2 and #3

File: lang/en/tool_userupsert.php (hunk 1 above)
Comment (dmitriim) [reply lang/en/tool_userupsert.php:37]: image

Comment (dmitriim) [reply lang/en/tool_userupsert.php:37]: image

**Comment by dmitriim** [comment]
@golenkovm you can merge bit by bit. Then you will get just new code in each of PR. They are all based on each other.

File: lang/en/tool_userupsert.php (hunk 1 above)
Comment (golenkovm) [reply lang/en/tool_userupsert.php:37]: Description of the fields, not the description of the admin settings

**Pull request merged**